packages/features/webhooks/lib/sendPayload.ts

```diff
@@ -50,6 +51,8 @@ export type WebhookDataType = CalendarEvent &
     createdAt: string;
     downloadLink?: string;
     paymentId?: number;
+    transcription?: TGetTranscriptAccessLink["transcription"];
+    recordingDownloadLink?: string;
```

Comment:
nit: How about we move recording related props in a separate type and add intersection with that here?
Also, I think if transcription is set recordingDownloadLink should also be set right? If that is the case we should be able to make them both required in `RecordingPayload` but make RecordingPayload optional